build/scripts/entrypoint-volume.sh

```diff
@@ -25,7 +25,6 @@ ls -la /checode/
 # Start the machine-exec component in background
 export MACHINE_EXEC_PORT=3333
 nohup /checode/bin/machine-exec --url "0.0.0.0:${MACHINE_EXEC_PORT}" &
-sleep 5
```

Comment:
@azatsarynnyy 
I've found this delay within investigation of another issue. I'm not sure why do we need it and if we can remove it

Reply:
@RomanNikitenko I agree, it looks redundant now.
I suppose the history of this `sleep` is something like this:

It was introduced within the initial implementation. At that time, Che-Code started requesting machine-exec during the Code server launch. Probably, some artificial delay was required because machin-exec wasn't ready to serve the clients.
Since I reworked communication with machine-exec and it's requested only after the corresponding built-in extension (che-terminal) has started, I believe we don't need that timeout anymore. As machine-exec has enough time to start now.